Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1287,6 +1290,89 @@ namespace PlayerCommands
 		PrintUserCmdText(client, L"|  <craftList> list <name/itemNr> - list materials necessary for selected item");
 	}
 
+	void UnlockRecipe(uint client, const wstring& args) 
+	{
+		PlayerBase* base = GetPlayerBaseForClient(client);
+
+		if (!checkBaseAdminAccess(base, client))
+		{
+			return;

```

Comment:
Missing the error message print "ERR No Admin Access" or somesuch.

Reply:
![image](https://github.com/DiscoveryGC/FLHook/assets/6022823/8bea63e1-2d33-4af7-a5f9-55c9a9745c3a)